src/examples/table/TableScroll.tsx

```diff
@@ -32,30 +33,25 @@ const rowData: IRow[] = Array.from(Array(100)).map((row, index) => ({
 }));
 
 const Example = () => (
-  <div style={{ overflowX: 'auto' }}>
+  <div style={{ maxHeight: 500, overflowY: 'auto' }}>
```

Comment:
Do we want to make this `overflow: auto` in both axis?
```suggestion
  <div style={{ maxHeight: 500, overflow: 'auto' }}>
```